### DISCUSSION TITLE
Error:  internet_connection_check

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#4641] [Questions] [answered]
ConnectException cURL error 6: Could not resolve host: www.edri.org (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for http://www.edri.org/

1. /var/www/html/3rdparty/guzzlehttp/guzzle/src/Handler/CurlFactory.phpline 158

    GuzzleHttp\Handler\CurlFactory::createRejection(
      "*** sensitive parameters replaced ***"
    )

2. /var/www/html/3rdparty/guzzlehttp/guzzle/src/Handler/CurlFactory.phpline 110

    GuzzleHttp\Handler\CurlFactory::finishError(
      [
        "GuzzleHttp\\Handler\\CurlHandler"
      ],
      "*** sensitive parameters replaced ***",
      [
        "GuzzleHttp\\Handler\\CurlFactory"
      ]
    )

 3.  /var/www/html/3rdparty/guzzlehttp/guzzle/src/Handler/CurlHandler.phpline 47

    GuzzleHttp\Handler\CurlFactory::finish(
      [
        "GuzzleHttp\\Handler\\CurlHandler"
      ],
      "*** sensitive parameters replaced ***",
      [
        "GuzzleHttp\\Handler\\CurlFactory"
      ]
    )

 4.   /var/www/html/3rdparty/guzzlehttp/guzzle/src/Middleware.phpline 137

    GuzzleHttp\Handler\CurlHandler->__invoke(
      "*** sensitive parameters replaced ***",
      "*** sensitive parameters replaced ***"
    )

  5.  /var/www/html/lib/private/Http/Client/DnsPinMiddleware.phpline 121

    GuzzleHttp\Middleware::GuzzleHttp\{closure}(
      "*** sensitive parameters replaced ***"
    )

6.    /var/www/html/3rdparty/guzzlehttp/guzzle/src/PrepareBodyMiddleware.phpline 35

    OC\Http\Client\DnsPinMiddleware->OC\Http\Client\{closure}(
      "*** sensitive parameters replaced ***"
    )

 7.   /var/www/html/3rdparty/guzzlehttp/guzzle/src/Middleware.phpline 31

    GuzzleHttp\PrepareBodyMiddleware->__invoke(
      "*** sensitive parameters replaced ***",
      "*** sensitive parameters replaced ***"
    )

  8.  /var/www/html/3rdparty/guzzlehttp/guzzle/src/RedirectMiddleware.phpline 71

    GuzzleHttp\Middleware::GuzzleHttp\{closure}(
      "*** sensitive parameters replaced ***"
    )

  9.  /var/www/html/3rdparty/guzzlehttp/guzzle/src/Middleware.phpline 63

    GuzzleHttp\RedirectMiddleware->__invoke(
      "*** sensitive parameters replaced ***",
      "*** sensitive parameters replaced ***"
    )

 10.   /var/www/html/3rdparty/guzzlehttp/guzzle/src/HandlerStack.phpline 75

    GuzzleHttp\Middleware::GuzzleHttp\{closure}(
      "*** sensitive parameters replaced ***"
    )

 11.   /var/www/html/3rdparty/guzzlehttp/guzzle/src/Client.phpline 331

    GuzzleHttp\HandlerStack->__invoke(
      "*** sensitive parameters replaced ***",
      "*** sensitive parameters replaced ***"
    )

12.    /var/www/html/3rdparty/guzzlehttp/guzzle/src/Client.phpline 168

    GuzzleHttp\Client->transfer(
      "*** sensitive parameters replaced ***",
      "*** sensitive parameters replaced ***"
    )

 13.   /var/www/html/3rdparty/guzzlehttp/guzzle/src/Client.phpline 187

    GuzzleHttp\Client->requestAsync(
      "*** sensitive parameters replaced ***"
    )

 14.   /var/www/html/lib/private/Http/Client/Client.phpline 230

    GuzzleHttp\Client->request(
      "*** sensitive parameters replaced ***",
      "http://www.edri.org/",
      [
        "/mnt/ncdata/files_external/rootcerts.crt",
        30,
        [
          true
        ],
        [
          "Nextcloud Server Crawler",
          "gzip"
        ],
        true
      ]
    )

15.    /var/www/html/apps/settings/lib/SetupChecks/InternetConnectivity.phpline 85

    OC\Http\Client\Client->get(
      "http://www.edri.org/"
    )

 16.   /var/www/html/apps/settings/lib/SetupChecks/InternetConnectivity.phpline 68

    OCA\Settings\SetupChecks\InternetConnectivity->isSiteReachable(
      "www.edri.org"
    )

  17.  /var/www/html/lib/private/SetupCheck/SetupCheckManager.phpline 49

    OCA\Settings\SetupChecks\InternetConnectivity->run()

18.    /var/www/html/apps/settings/lib/Controller/CheckSetupController.phpline 181

    OC\SetupCheck\SetupCheckManager->runAll()

 19.   /var/www/html/lib/private/AppFramework/Http/Dispatcher.phpline 230

    OCA\Settings\Controller\CheckSetupController->check()

2.    /var/www/html/lib/private/AppFramework/Http/Dispatcher.phpline 137

    OC\AppFramework\Http\Dispatcher->executeController(
      [
        "OCA\\Settings\\Controller\\CheckSetupController"
      ],
      "check"
    )

21.    /var/www/html/lib/private/AppFramework/App.phpline 184

    OC\AppFramework\Http\Dispatcher->dispatch(
      [
        "OCA\\Settings\\Controller\\CheckSetupController"
      ],
      "check"
    )

22.    /var/www/html/lib/private/Route/Router.phpline 315

    OC\AppFramework\App::main(
      "OCA\\Settings\\Controller\\CheckSetupController",
      "check",
      [
        "OC\\AppFramework\\DependencyInjection\\DIContainer"
      ],
      [
        "settings.CheckSetup.check"
      ]
    )

  23.  /var/www/html/lib/base.phpline 1069

    OC\Route\Router->match(
      "/settings/ajax/checksetup"
    )

  24  /var/www/html/index.phpline 39

    OC::handleRequest()

I recently upgrade my Nextcloud All-In-One (AIO) installation from version 27 to version 28. However, I encountered this issue after upgrade this version. 

### System Details:
**OS:**  Debian GNU/Linux
**VERSION:** 10 (buster)
**Docker Version:** 20.10.21

**Now I need help to solve this issue. **

**Comment by szaimen** [comment, marked as answer]
Hi, it looks like your dns resolution ad/or internet is not working
